src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -534,7 +537,10 @@ export default class MutationMapper<
 
         return (
             <FilterResetPanel
-                resetFilters={() => dataStore.resetFilters()}
+                resetFilters={() => {
+                    dataStore.resetFilters();
+                    this.driverVusVusSelectorComponent?.reset();
```

Comment:
Ideally we should override the default filter applier for mutation type and handle driver vs vus logic in the filter applier function, too. We can then derive everything from the filter itself. That would eliminate the need for a custom reset function.